### PR TITLE
RDoc-580

### PR DESCRIPTION
**Pull request by IdanHaim**

### CODE REVIEW

File: Documentation/3.5/Raven.Documentation.Pages/server/scaling-out/replication/write-assurance.markdown [hunk 1]
```diff
@@ -18,9 +18,33 @@ You can also use parameterless version of the `WaitAsync` method:
...
+
+{CODE-BLOCK: csharp} 
+    using (var session = store.OpenSession()) 
+    { 
+        User user = new User(); 
+        session.Store(user); 
+        session.Advanced.WaitForReplicationAfterSave(replicas: 2, timeout: TimeSpan.FromSeconds(30)); 
+        session.saveChanges(); 
+    } 
+{CODE-BLOCK/}
```
Comment (ppekrol) [inline Documentation/3.5/Raven.Documentation.Pages/server/scaling-out/replication/write-assurance.markdown]: Avoid having a code in markdown file directly. Use code from *.cs file.

File: Documentation/3.5/Raven.Documentation.Pages/server/scaling-out/replication/write-assurance.markdown [hunk 2]
```diff
@@ -18,9 +18,33 @@ You can also use parameterless version of the `WaitAsync` method:
...
+
+Like the WaitAsync You can also use parameterless version to the method:
+
+{CODE-BLOCK: csharp} session.Advanced.WaitForReplicationAfterSave(); {CODE-BLOCK/}
```
Comment (ppekrol) [inline Documentation/3.5/Raven.Documentation.Pages/server/scaling-out/replication/write-assurance.markdown]: Same here

**Comment by IdanHaim** [comment]
I know I wrote this way because 
session.Advanced.WaitForReplicationAfterSave
is not yet updated in raven.client

**Comment by ravendb** [comment]
If it isn't, you need to update it so the code will have it.

*Hibernating Rhinos Ltd  *

Oren Eini* l CEO l *Mobile: + 972-52-548-6969

Office: +972-4-622-7811 *l *Fax: +972-153-4-622-7811

**Pull request closed without merging**